microsoft/surface/common/kernel/linux-6.7.x/default.nix

```diff
@@ -7,14 +7,14 @@ let
 
   cfg = config.microsoft-surface;
 
-  version = "6.6.13";
+  version = "6.7.6";
   kernelPatches = surfacePatches {
     inherit version;
     patchFn = ./patches.nix;
   };
   kernelPackages = linuxPackage {
     inherit version kernelPatches;
-    sha256 = "sha256-iLiefdQerU46seQRyLuNWSV1rPgVzx3zwNxX4uiCwLw=";
+    sha256 = "";
```

Comment:
Empty hash should be `sha256-5InsDhNw0Im0RtVlre16aYCT0rfEEioY8h7bbvk9N9M=`

Reply:
Any chance we can bump this to `arch-6.7.9-1`? That pulls in https://github.com/linux-surface/linux-surface/commit/852fcb1c2f7be2ac8e115984432761e0edaa7801 which has improvements to the fan profiles on the SP9.

To cherry-pick (on top of @tracteurblinde 's commit); cf462d6d1bb8d3c023ccabdb56e5c0af1a1802ed

Reply:
> Any chance we can bump this to `arch-6.7.9-1`? That pulls in [linux-surface/linux-surface@852fcb1](https://github.com/linux-surface/linux-surface/commit/852fcb1c2f7be2ac8e115984432761e0edaa7801) which has improvements to the fan profiles on the SP9.
> 
> To cherry-pick (on top of @tracteurblinde 's commit); [cf462d6](https://github.com/NixOS/nixos-hardware/commit/cf462d6d1bb8d3c023ccabdb56e5c0af1a1802ed)

not only that but also finally has working cameras again which would be nice to have